sdk/include/reactos/debug.h

```diff
@@ -18,6 +18,10 @@
 #define __RELFILE__ __FILE__
 #endif
 
+#ifdef NDEBUG
+#undef DBG
+#endif
```

Comment:
Why doing this? `DBG` is the general macro that tells whether a program is compiled in debug mode. `˙NDEBUG` means, that we don't want the debug macros to show debug output when using debug.h.

Reply:
Some codes manually define `NDEBUG` and disable the debugging output... I followed this manner.

https://github.com/reactos/reactos/blob/47f6745bcdfb833b0311e052f04e6cd5634660eb/win32ss/gdi/ntgdi/freetype.c#L36-L37

https://github.com/reactos/reactos/blob/47f6745bcdfb833b0311e052f04e6cd5634660eb/win32ss/gdi/ntgdi/bitblt.c#L10-L11

Reply:
Yes but this means, that if they do an explicit `DPRINT1` (not `DPRINT`) then the DPRINT1 output will be output, which is what is expected. When `NDEBUG` is not defined, which is e.g. usually done when testing something locally, both `DPRINT` and `DPRINT1` will be output (this is one thing I forgot to tell in the previous message). But in all cases, the generic `DBG` define symbol still remains as it is (untouched).